docs/source/api_reference/torch_layers.csv

```diff
@@ -40,3 +40,4 @@ MolGANAggregationLayer,`ref <https://arxiv.org/abs/1805.11973>`_, MolGan
 DTNNEmbedding, ref`<https://arxiv.org/abs/1609.08259>`_, DTNNModel
 DTNNStep, ref`<https://arxiv.org/abs/1609.08259>`_, DTNNModel
 DTNNGather, ref`<https://arxiv.org/abs/1609.08259>`_, DTNNModel
+GlobalMessagePassing, `ref <https://arxiv.org/pdf/2011.07457.pdf>`_, Global Message Passing Neural Network
```

Comment:
Change the reference to the `https://arxiv.org/abs/2011.07457` instead of the pdf version to maintain consistency with the rest.

Reply:
okay, I'll do